### PR TITLE
Classify paramref and typeparamref in Quick Info

### LINKED ISSUE #33546: VS IntelliSense display paramref elements same as regular text
**Version Used**: N/A

**Steps to Reproduce**: 

1. Use VS to display IntelliSense pop-up text.

**Expected Behavior**:
The text within paramref tags should be rendered in different font from regular text to distinguish as a parameter which is not translated by-design. 

**Actual Behavior**:
The text within paramref tags are rendered the same as translated text, so it appears like we forgot to translate that text.

Screenshots available in email thread "RE: interesting ref doc loc rendering 'issue'.

### PR DESCRIPTION
**Pull request by sharwell**
Fixes #33546

### CODE REVIEW

File: src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs [hunk 1]
```diff
@@ -41,21 +42,48 @@ internal static class IntellisenseQuickInfoBuilder
...
+                {
+                    if (isFirstElement)
+                    {
+                        isFirstElement = false;
+                        firstLineElements.Add(element);
+                    }
+                    else
+                    {
+                        elements.Add(element);
+                    }
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs]: it's a draft, so feel free to ask me to wait.  However, i was surprised to see cahnges like this here...  are they really necessary if we're just classifying paramrefs?

File: src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs [hunk 2]
```diff
@@ -155,17 +175,18 @@ private static void AppendTextFromNode(FormatterState state, XNode node, Compila
...
                 return;
             }
             else if (name == "paramref" ||
                      name == "typeparamref")
             {
+                var kind = name == "paramref" ? SymbolDisplayPartKind.ParameterName : SymbolDisplayPartKind.TypeParameterName;
                 foreach (var attribute in element.Attributes())
                 {
-                    AppendTextFromAttribute(state, element, attribute, attributeNameToParse: "name");
+                    AppendTextFromAttribute(state, element, attribute, attributeNameToParse: "name", kind);
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs]: i would have expected to just see this.

**Comment by CyrusNajmabadi** [comment]
> Builds on #33028

Consider not doing that.  they seem like orthogonal changes.

File: src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs (hunk 1 above)
Comment (sharwell) [reply src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs]: Only the last commit is specific to this pull request. The first comment links to the main PR.

**Comment by sharwell** [comment]
> Consider not doing that. they seem like orthogonal changes.

I need the other PR so I can write tests for this. I'm targeting it to 16.1P1 and am planning to remove the draft label from this PR after it's merged.

File: src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs (hunk 2 above)
Comment (sharwell) [reply src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs]: The last commit is close to just this. 👍

**Comment by CyrusNajmabadi** [comment]
Gotcha.  Last commit seems good.  That part is LGTM :)  The stuff preceding that i ahven't looked at yet :)

While you're up, how about some merging of my PRs ;-)

**Pull request merged**